Compiler/FrontEnd/ModelicaBuiltin.mo

```diff
@@ -3869,6 +3869,16 @@ annotation(
 </html>"));
 end GC_expand_hp;
 
+function GC_set_max_heap_size
```

Comment:
Would be nice to follow the API naming style i.e., camelCase.

Reply:
Then it should be GC.setMaxHeapSize, but that doesn't quite work. At least it follows the existing style for GC_expand_hp